spec/support/fixtures/adobe_campaign_parsed_json_fixtures.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/ModuleLength
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/LineLength
+module AdobeCampaignParsedJsonFixtures
+  def adobe_campaign_profile
+    {
+      "PKey" => "@EJcyhZ11jryZtasdfIAHR2c8foWIzASt5-c9p3FShoQR8nTDjEp7xY55lHFPiEffYnCm8nsIW0zIvoVt0-3zrQ_GjTs",
+      "age" => 0,
+      "birthDate" => "",
+      "blackList" => false,
+      "blackListEmail" => false,
+      "blackListFax" => false,
+      "blackListMobile" => false,
+      "blackListPhone" => false,
+      "blackListPostalMail" => false,
+      "blackListPushnotification" => false,
+      "created" => "2018-04-05 23:20:14.989Z",
+      "cryptedId" => "2sEnzQAmEYxZNM1DU1HEsODdTPpn1REzNGJpWqfQObpHg4aBDm9IPtV8qGBSC1L/0EjWWg==",
+      "cusAccountName" => "",
+      "cusAccountNumber" => "",
+      "cusAccountTitle" => "",
+      "cusAccountType" => "",
+      "cusAddressName" => "",
+      "cusBdayofMonth" => 0,
+      "cusBirthMonth" => 0,
+      "cusBirthYear" => 0,
+      "cusBusinessPhone" => "",
+      "cusCampusName" => "",
+      "cusCompany" => "",
+      "cusContactId" => "",
+      "cusDEPTNAME" => "",
+      "cusDonor" => "N",
+      "cusDonorClass" => "",
+      "cusEmployee" => "N",
+      "cusEventAttendeelink" => {
+        "href" => "https://mc.adobe.io/cru/campaign/profileAndServicesExt/profile/@EJcyhZ11jryZtasdfIAHR2c8foWIzASt5-c9p3FShoQR8nTDjEp7xY55lHFPiEffYnCm8nsIW0zIvoVt0-3zrQ_GjTs/cusEventAttendeelink/",
+      },
+      "cusFinanceAccountNumber" => "",
+      "cusGCXUrlslinktoProfile" => {
+        "href" => "https://mc.adobe.io/cru/campaign/profileAndServicesExt/profile/@EJcyhZ11jryZtasdfIAHR2c8foWIzASt5-c9p3FShoTry94ItJPgAbfCOQNN4SKxKc2ArQdTul7snRO40-8xGu6pk8SEhPPQuN0FIWm1eszvWUtS/cusGCXUrlslinktoProfile/",
+      },
+      "cusGlobalID" => "",
+      "cusGraduationDate" => "",
+      "cusJOBTITLE" => "",
+      "cusLink" => {
+        "href" => "https://mc.adobe.io/cru/campaign/profileAndServicesExt/profile/@EJcyhZ11jryZtasdfIAHR2c8foWIzASt5-c9p3FShoQR8nTDjEp7xY55lHFPiEffYnCm8nsIW0zIvoVt0-3zrQ_GjTs/cusLink/",
+      },
+      "cusLink2" => {
+        "href" => "https://mc.adobe.io/cru/campaign/profileAndServicesExt/profile/@EJcyhZ11jryZtasdfIAHR2c8foWIzASt5-c9p3FShoTomQPm9e3ETJ_s2pUrH5DVPZuk_Z0W41Q6y_gWBZNXq0tVd1bKDLzkB92oTy7ZdPgYWNvX/cusLink2/",
+      },
+      "cusMinistryCode" => "",
+      "cusMinistryCodeGR" => "",
+      "cusMissionHubUser" => "Y",
```

Comment:
You should probably change all `MissionHub` references to `GodTools` (or `MobileContent`).
@ckozus

Reply:
@hlbraddock done!